### PR TITLE
docs: added delete verb section

### PR DESCRIPTION
**Pull request by natt-n**
**What I did**

Added the 'delete' verb section.

### CODE REVIEW

**Review by JeremyTubongbanua — APPROVED** [review]

**Review by JeremyTubongbanua — CHANGES_REQUESTED** [review]
In #81 , I added these headers to each verb (Synopsis, Syntax, Example, Response, Description, and Options).

Do you think we could do the same for the `delete` verb too?

**Review by gkc — DISMISSED** [review]

**Comment by gkc** [comment]
You need to resolve conflicts, @JeremyTubongbanua

**Comment by JeremyTubongbanua** [comment]
> In #81 , I added these headers to each verb (Synopsis, Syntax, Example, Response, Description, and Options).
> 
> Do you think we could do the same for the `delete` verb too?

Done in https://github.com/atsign-foundation/at_protocol/pull/82/commits/ea42786a7ee854e630b7ec3187542372ae9855a7

**Review by JeremyTubongbanua — DISMISSED** [review]

**Comment by JeremyTubongbanua** [comment]
Also added the following in [commit](https://github.com/atsign-foundation/at_protocol/commit/ea42786a7ee854e630b7ec3187542372ae9855a7) 

- Syntax header
- Options table


Image Preview:
image

File: specification/at_protocol_specification.md [hunk 1]
```diff
@@ -912,6 +912,47 @@ The `llookup` verb should be used to fetch the value of the key in the owners at
...
+
+**Synopsis:**
+
+The `delete` verb should be used to delete an atKey in the atServer. Only authenticated atSigns can use the `delete` verb. 
+
+**Syntax:**
+
+The following is the regex of the `delete` verb:
+
+```^delete:(?:cached:)?((?:public:)|(@(?<for@sign>[^@:\s]-):))?(?<atKey>[^:]((?!:{2})[^@])+)@(?<@sign>[^@\s]+)$```
```
Comment (gkc) [inline specification/at_protocol_specification.md]: That is not the latest regex. See https://github.com/atsign-foundation/at_tools/blob/trunk/packages/at_commons/lib/src/verb/syntax.dart#LL66C3-L72C12

**Review by gkc — CHANGES_REQUESTED** [review]

File: specification/at_protocol_specification.md (hunk 1 above)
Comment (JeremyTubongbanua) [reply specification/at_protocol_specification.md]: See [af47555](https://github.com/atsign-foundation/at_protocol/pull/82/commits/af4755511dca6604b03557b5bcc18227c46e47d0)

**Review by gkc — APPROVED** [review]

**Pull request merged**